File: AoC2019/Python3/day1.py
import math

#puzzle setup
with open('./day1in.txt','r') as pInput:
    pInput = pInput.read().splitlines()

#do stuff
mysum = 0

for x in pInput:
    while int(x) > 5:
        x = math.floor(int(x) / 3) - 2
        mysum += x

print(mysum)

File: AoC2019/JavaScript/day1.js
//puzzle setup
const fs = require('fs');
input = fs.readFileSync('../day1in.txt','utf8');
arr = input.split(/\r?\n/);

//do stuff
sum = 0;
for (i in arr)
{
    sum += Math.floor(arr[i]/3) - 2;
}
console.log(sum);